_posts/2017-03-05-artsy-technology-stack-2017.md

```diff
@@ -49,11 +49,15 @@ We have three major CMS projects:
 [Writer]
 [Genome tool]
 
-## Analytics
+## Data Pipeline
+
```

Comment:
I would add something general somewhere like _Data generally flows from consumer applications and services to Redshift._